app/routing/ArrivalsRouter.scala

```diff
@@ -42,4 +44,10 @@ class ArrivalsRouter @Inject() (
       case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE) => v2Arrivals.createArrivalNotification()
       case _                                                    => v1Arrivals.createArrivalNotification()
     }
+
+  def getArrivalsForEori(updatedSince: Option[OffsetDateTime] = None): Action[Source[ByteString, _]] = route {
+    case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE) => v2Arrivals.getArrivalsForEori(updatedSince)
```

Comment:
Test cases for ArrivalsRouter seems missing for this method implementation in ArrivalsRouterSpec